documentation/modules/rn-2.6.adoc

```diff
@@ -0,0 +1,294 @@
+// 
+// .. Block Migrations from OpenShift versions before 4.13
+// 
+// This does not look like a feature. Is this a resolved issue?
+
+.. Secure connections to remote OpenShift clusters
+
+Connections to remote OpenShift clusters can be secures with CA certificates. link:https://issues.redhat.com/browse/MTV-728[MTV-728]
+
+// Is this a feature or Resolved issue?
+// .. Source and storage endpoints are not selected automatically
+// 
+// Source network and source storage anre not seleted automatically when creating an OCP to OCP plan. link:https://issues.redhat.com/browse/MTV-729[MTV729] 
+
+.Console Plugin
+
+This is a collection of new features for the {project-short} Console plugin. link:https://issues.redhat.com/browse/MTV-736[(MTV-736)]
+
+..MTV-615
+
+..MTV-737
+
+..MTV-245
+
+..MTV-365
+
+..MTV-449
+
+..MTV-616
+
+..MTV-773
+
+..MTV-724
+
+..MTV-766
+
+..MTV-768
+
+..MTV-770
+
+..MTV-772
+
+..MTV-775
+
+..MTV-779
+
+..MTV-792
+
+..MTV-793
+
+..MTV-797
+
+..MTV-867
+
+.OVA Provider
+
+OVA provider. link:https://issues.redhat.com/browse/MTV-690[(MTV-690)]
+
+..MTV-759
+
+..MTV-848
+
+..MTV-624
+
+..MTV-681
+
+..MTV-680
+
+..MTV-689
+
+..MTV-733
+
+..MTV-669
+
+..MTV-715
+
+..MTV-714
+
+..MTV-696
+
+[id="known-issues-26_{context}"]
+== Known issues
+
+This release has the following known issues:
+
+.Deleting migration plan does not remove temporary resources
+
+Deleting a migration plan does not remove temporary resources such as importer pods, conversion pods, config maps, secrets, failed VMs and data volumes. You must archive a migration plan before deleting it to clean up the temporary resources. link:https://bugzilla.redhat.com/show_bug.cgi?id=2018974[(BZ#2018974)]
+
+.Unclear error status message for VM with no operating system
+
+The error status message for a VM with no operating system on the *Plans* page of the web console does not describe the reason for the failure. link:https://bugzilla.redhat.com/show_bug.cgi?id=2008846[(BZ#22008846)]
+
+.Migration of virtual machines with encrypted partitions fails during conversion
+
+vSphere only: Migrations from {rhv-short} and OpenStack do not fail, but the encryption key may be missing on the target {ocp} cluster.
+
+
+.Migration fails during precopy/cutover while a snapshot operation is performed on the source VM
+
+Warm migration from {rhv-short} fails if a snapshot operation is performed on the source VM. If a user performs a snapshot operation on the source VM at the time when a migration snapshot is scheduled, the migration fails instead of waiting for the user’s snapshot operation to finish. link:https://issues.redhat.com/browse/MTV-456[(MTV-456)]
+
+.Unable to schedule migrated VM with multiple disks to more than one storage classes of type hostPath
+
+When migrating a VM with multiple disks to more than one storage classes of type `hostPath`, it might happen that a VM cannot be scheduled. Workaround: Use shared storage on the target {ocp} cluster.
```

Comment:
`it might happen`

```suggestion
When migrating a VM with multiple disks to more than one storage classes of type `hostPath`, a VM might not  be scheduled. Workaround: Use shared storage on the target {ocp} cluster.
```